indexer/ai/src/main/java/com/google/cloudsearch/ai/Mapping.java

```diff
@@ -0,0 +1,31 @@
+package com.google.cloudsearch.ai;
+
+import java.util.List;
+
+class Mapping {
+    private List<String> PropertyNames;
```

Comment:
why this is a list of string ?

Reply:
I was providing the option of mapping one skill output to multiple properties in case when we have multiple objects in the schema. Then the skill is applied on different data corresponding to the particular object and mapped to the property.

Reply:
Okay, I think I will change it to String, as we are providing  options for only one object.